src/FixedDataTableTranslateDOMPosition.js

```diff
@@ -20,9 +22,18 @@ function FixedDataTableTranslateDOMPosition(/*object*/ style, /*number*/ x, /*nu
     style.left = x + 'px';
     style.top = y + 'px';
   } else {
+    if (BrowserSupportCore.hasCSSTransforms() && Locale.isRTL()) {
+      x *= Locale.DIR_SIGN();
```

Comment:
if we are using DIR_SIGN() to assign, then we don't need the `Locale.isRTL()` check above